triton-cli/test-vectors/simple.tasm

```diff
@@ -0,0 +1,24 @@
+push 10
+push 100
+push 10000
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+pop 1
+push 5
+swap 3
+halt
```

Comment:
nit: [POSIX says that a line must end with `\n`](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline). That will remove github's the scary-looking “🚫”. :smiley: